public/scripts.js

```diff
@@ -164,7 +101,47 @@ $("div").children('input').on("keypress", function () {
 
 /*---Delete row from table---*/
 $('table').on("click", 'button', function () {
-  console.log($(this).index(".icon-button"));
-  simulators.splice(($(this).index(".icon-button")), 1);
-  buildTable(simulators);
+  ajaxReq('DELETE', '/api/simulators/' + $(this).data('simulator_id'))
+    .done(function (res) {
+      ajaxRexGet();
+    })
 });
+
+function sumprice(build_array) {
+  var sum = 0;
+  for (var i = 0; i < build_array.length; i++) {
+    sum += build_array[i].price;
+  }
+  return sum;
+}
+
+function formatCurrency(fprice) {
+  return '$' + fprice.toFixed(0).replace(/(\d)(?=(\d\d\d)+(?!\d))/g, "$1,");
+}
+
+function checkedID(tmp_type_number) {

```

Comment:
Ez totál felesleges, megnézi a post, hogy van-e ilyen és úgy is visszatér 400-zal, és azt lehet kezelni egy .fail chaininggel...amúgy is kellene vmi basic .fail, ami kiírja a juzernek, ha nem műxik a kapcsolat a szerverrel...